Sources/Datadog/DatadogCore/Storage/Reading/FileReader.swift

```diff
@@ -62,21 +62,25 @@ internal final class FileReader: Reader {
             // get event blocks only
             .compactMap {
                 switch $0.type {
-                case .event:
-                    return $0.data
+                case .event, .eventMetadata:
+                    return $0
                 }
             }
```

Comment:
I guess we don't need this filter anymore, we can return all blocks 👍

Reply:
removed.